craft_application/services/lifecycle.py

```diff
@@ -210,21 +210,14 @@ def _init_lifecycle_manager(self) -> LifecycleManager:
         emit.debug(f"Initialising lifecycle manager in {self._work_dir}")
         emit.trace(f"Lifecycle: {repr(self)}")
 
+        project = self._services.get("project")
```

Comment:
```suggestion
        project_service = self._services.get("project")
```

it was confusing further down

Reply:
Good catch. 

https://github.com/canonical/craft-application/pull/872/commits/262a4bab2db618d5d3c27916a36acaaa1f14d40b